include/cuco/detail/static_multimap/static_multimap.inl

```diff
@@ -906,10 +909,22 @@ template <typename Key,
 std::size_t static_multimap<Key, Value, Scope, Allocator, ProbeSequence>::get_size(
   cudaStream_t stream) const noexcept
 {
-  auto begin  = thrust::make_transform_iterator(raw_slots(), detail::slot_to_tuple<Key, Value>{});
-  auto filled = cuco::detail::slot_is_filled<Key>{get_empty_key_sentinel()};
+  using namespace experimental::detail;
```

Comment:
I'm always a bit cautious about pulling the `detail` namespace into the current scope since it can result in naming collisions which are hard to figure out during debugging.

Reply:
shortcut removed